src/applications/personalization/profile360/constants/vet360.js

```diff
@@ -32,3 +32,15 @@ export const PHONE_TYPE = {
   faxNumber: 'FAX',
   homePhone: 'HOME',
 };
+
+export const ANALYTICS_FIELD_MAP = {
+  primaryTelephone: 'primary-telephone',
+  alternateTelephone: 'alternative-telephone',
+  homePhone: 'home-telephone',
+  mobilePhone: 'mobile-telephone',
+  workPhone: 'work-telephone',
+  faxNumber: 'fax-telephone',
+  email: 'email',
+  mailingAddress: 'mailing-address',
+  residentialAddress: 'home-address'
+};
```

Comment:
Nice! Not that we need to worry about it right now, but I was wondering if a full-out mapping might be beneficial, like:

```
const FIELDS = {
  [FIELDNAMES.PRIMARY_PHONE]: {
    vet360PropertyName: 'primaryPhone',
    analyticsName: 'primary-telephone',
    label: 'Primary phone number'
  }
}
```

But I could also see that being overkill.